### DISCUSSION TITLE
Additional source for callsign lookup

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#1590] [Ideas / Feature Requests]
Is a possibility to add another source for callsign lookup?
I mean qrz.ru database, API is open to requests and documented (https://qrz.ru/help/api/xml).
I'm new in php-web programming, so i added only button and menu item for manual open profile on qrz.ru.
![2022-09-02_17-27-40](https://user-images.githubusercontent.com/16667173/188171280-6eb3131f-72bd-43c3-a427-98107d7bfba7.png)
![2022-09-02_17-27-05](https://user-images.githubusercontent.com/16667173/188171282-3762f203-c0d5-4459-bd34-d3c10462af3d.png)

**Comment by AndreasK79** [comment]
I see no reason why this couldn't be added. We just need someone to do it.

**Reply by R1BLH** [reply]
I am not a software developer, I am a radio engineer. :)
I could post here in the comments what I already did (the results are screenshots above), but I'm really a dummy in web programming. :)
So id did static burttons for open callsign profile, but in do not know how to make automatic lookup on the entering callsign in the QSO form.

**Comment by magicbug** [comment]
This could be added just needs its on PHP library created to handle how lookups are carried out, its a good suggestion but something that will need to wait a little while
